src/tool_writeout.c

```diff
@@ -331,6 +469,24 @@ void ourWriteOut(CURL *curl, struct OutStruct *outs, const char *writeinfo)
               case VAR_STDERR:
                 stream = stderr;
                 break;
+              case VAR_CONTENT:
+                fflush(outs->buffer_stream);
+                if(NULL != outs->bodybuf) {
+                  if('\"' == *(ptr-3)) {
```

Comment:
please explain what this is? This is not documented.

is dereffing the pointer at -3 really always safe? It seems that could be before the string?

Reply:
Already resolve.